include/boost/gil/image.hpp

```diff
@@ -100,6 +100,19 @@ class image {
                                            , _allocated_bytes( img._allocated_bytes ) {
        allocate_and_copy(img.dimensions(),img._view);
     }
+    
+    image(image&& img) noexcept {
+        _view = img._view;
```

Comment:
These are copy/move assignments, not constructions.

Reply:
Thanks for the review, would this be better:

```c++
   image(image&& img) :
       _view(img._view),
       ...
```

Reply:
I made the suggested modifications.